src/test/java/org/folio/rest/impl/EntitiesCrudBasicsTest.java

```diff
@@ -210,17 +211,21 @@ public void testPostRecordForUnprocessibleEntity(TestEntities testEntity) {
   }
 
   @ParameterizedTest
-  @MethodSource("getTestEntitiesPostWithoutTransaction")
-  public void testPostRecordServerError(TestEntities testEntity) {
+  @MethodSource("getTestEntitiesWithPostEndpoint")
+  public void testPostRecordServerError(TestEntities testEntity) throws IOException {
     logger.info("=== Test create {} record - Internal Server Error ===", testEntity.name());
 
     Headers headers = prepareHeaders(X_OKAPI_URL, ERROR_X_OKAPI_TENANT);
-    verifyPostResponse(testEntity.getEndpoint(), testEntity.getMockObject(), headers, APPLICATION_JSON,
+    JsonObject record = testEntity.getMockObject();
+    if(testEntity.equals(TestEntities.ORDER_TRANSACTION_SUMMARY)) {
```

Comment:
This `if` can be removed because `ORDER_TRANSACTION_SUMMARY.jsonPathToObject` is fixed.

Reply:
overlooked this. Fixed